HHS/Head-Start-TTADP
# [TTAHUB-3040] Resolve OWASP warnings

## Description of change
Resolve OWASP generated warnings in the CI around the content security policy by adding new configuration and self-hosting a font we previously linked from Google. Also disallowed the "x-powered-by" header to stop easy sniffing of our platform code.

Note that we are currently unable to resolve one medium issue around the style-src CSP. Plotly still injects CSS in an manner considered unsafe. [issue](https://github.com/plotly/plotly.js/issues/2355), [fix awaiting merge](https://github.com/plotly/plotly.js/pull/6239)

## How to test
Confirm that [all the "medium" errors are resolved in the dynamic security scan.](https://output.circle-artifacts.com/output/job/cee4dab4-ae2e-4a99-b4b1-72151acef533/artifacts/0/reports/owasp_report.html) except for the 1 that cannot be resolved ("style-src contains unsafe-inline"

Confirm that assets are still loaded as expected and the site still functions, etc. This PR is deployed to sandbox for easier testing.

## Issue(s)
* https://jira.acf.gov/browse/TTAHUB-3039
* https://jira.acf.gov/browse/TTAHUB-3040


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
